### DISCUSSION TITLE
New Project Folder Structure

### DISCUSSION THREAD

**Discussion by dougransom** [RussAbbott/pylog#6] [General]
@imjaypatel12's PR https://github.com/RussAbbott/pylog/pull/1 looks like a good idea, to restructure into src/pylog, test, etc.  It is a good convention and many projects follow it.     

Unfortunately there are some merge conflicts because work has been done on the master branch after Jay's versions were moved.   

How do we get  this PR merged?  Or is it better to start over, and just move the files into the new structure on master?  

It would be best to get this done sooner than later if we are to have someone working on the documentation.

**Comment by imjaypatel12** [comment]
I can work on resolving the conflicts and bring my branch up to the latest
state. After that, it will let us merge it to main repo.

I am on vacation till 30th. Once I’m back, I’ll work on resolving this.

- Jay

**Reply by dougransom** [reply]
@imjaypatel12 nudge.

**Comment by RussAbbott** [comment]
Thanks Jay,

Hope you had a good vacation.  My input is so flooded with political
solicitations that I'm having trouble going through my email. Looks like it
will continue like this until the election.

-- Russ Abbott
Professor Emeritus, Computer Science
California State University, Los Angeles

**Comment by imjaypatel12** [comment]
I have updated my PR. It can now be merged. Thanks!

**Comment by dougransom** [comment]
Thanks Jay.  Merged in, made some minor edits to instructions, pyproject.toml, switched to flit from setuptools for the build step, added github action to build/publish on release (not working yet).
Minor update, docs, LICENSE, pyroject.toml (#7) * ***@***.***<https://github.com/RussAbbott/pylog/commit/17a89b7fd54447a3c0e761ed6a05827fb0730a98>
Doug Ransom
***@***.***
